config/locales/details.yml

```diff
@@ -14,6 +14,16 @@ en:
       uploaded_files_header: Uploaded files
       no_uploaded_files: No files uploaded
       having_problems: I am having trouble uploading my documents
+      
+      explanation_html:
+        <p><strong>What to do next</strong></p>
+        <ul class="list list-bullet">
+          <li>Call the tax tribunal on 0300 123 1024 for help (see <a href="https://www.gov.uk/call-charges">call charges</a>).</li>
```

Comment:
We have this phone number defined in config/application.rb as config.tax_tribunal_phone, so I think we should reuse that here, rather than putting it again.